sdk/mobile-center-distribute/src/test/java/com/microsoft/azure/mobile/distribute/DistributeHttpTest.java

```diff
@@ -79,6 +79,34 @@ public void onBeforeCalling() throws Exception {
     }
 
     @Test
+    public void onBeforeCallingWithoutToken() throws Exception {
```

Comment:
You already tested this in the above method. Isn't it a duplicate test?
Look at line 60, it doesn't have token in header.